packages/plugin-graphql/src/schema/graph.js

```diff
@@ -66,7 +66,7 @@ const getParsedHeadingsFromPage = (tableOfContents = [], headingLevel) => {
 };
 
 const getPagesFromGraph = async (root, query, context) => {
-  return Promise.resolve(context.graph);
+  return context.graph;
```

Comment:
this is probably fine, but as mentioned, maybe better to take all these `Promise` clean up tasks into their own PR?